doc/quoting-issues-with-powershell.md

```diff
@@ -169,10 +169,18 @@ Command arguments: ['{"key": "value"}', '--debug']
 
 This issue is tracked at https://github.com/PowerShell/PowerShell/issues/1995#issuecomment-539822061
 
-## Workaround: use file input
+## Best practice: use file input
 
-You may use CLI's `@<file>` convention to load from a file to bypass the shell's interpretation mechanisms:
+For complex arguments like JSON string, the best practice is to use Azure CLI's `@<file>` convention to load from a file to bypass the shell's interpretation mechanisms.
+
+Note that At symbol (`@`) is [splatting operator](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_splatting) in PowerShell, so it should be surrounded by quotes.
+
+```powershell
+az debug echo "@test.json"
+```
+
+You may also use `@-` to read from `stdin`:
 
 ```powershell
-az ad app create --display-name my-native --native-app --required-resource-accesses @manifest.json
+Get-Content -Path test.json | az debug echo "@-"
```

Comment:
Is there an example that shows how to use it with Azure CLI?

Reply:
`az debug echo` is hidden (not user-visible).